<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sudokuGUIClass</class>
 <widget class="QMainWindow" name="sudokuGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>414</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>sudokuGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 247, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>139</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="panelLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="optionsLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>45</number>
       </property>
       <property name="rightMargin">
        <number>45</number>
       </property>
       <item>
        <widget class="QRadioButton" name="simpleOption">
         <property name="text">
          <string>简单</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optionButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="normalOption">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Agency FB&quot;;</string>
         </property>
         <property name="text">
          <string>入门</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optionButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hardOption">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;</string>
         </property>
         <property name="text">
          <string>困难</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">optionButtons</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);
font: 75 9pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>411</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sudokuLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menuLayout">
     <item>
      <widget class="QPushButton" name="informButton">
       <property name="text">
        <string>提示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="recordLayout">
     <item>
      <widget class="QLabel" name="timelabel">
       <property name="text">
        <string>当前用时：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="timeLcd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sudokuGUI.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="optionButtons"/>
 </buttongroups>
</ui>
